pkg/user/account.go

```diff
@@ -81,3 +82,21 @@ func QueryAccount(ctx context.Context, conn *grpc.ClientConn, registry codectype
 	accNum, seqNum = acc.GetAccountNumber(), acc.GetSequence()
 	return accNum, seqNum, nil
 }
+
+// QueryAccountBalance fetches the balance of a specific denomination for an account.
+func QueryAccountBalance(ctx context.Context, conn *grpc.ClientConn, registry codectypes.InterfaceRegistry, address types.AccAddress, denom string) (types.Coin, error) {
```

Comment:
used to determine if we need to send a single utia to an account to create it or if it already exists